tests/e2e/testutils.go

```diff
@@ -122,23 +115,32 @@ func FetchNodeExternalIP(nodename string) (string, error) {
 	nodeip := strings.TrimSpace(ip[1])
 	return nodeip, nil
 }
-func FetchIngressIP(kubeconfig string) ([]string, error) {
-	cmd := "kubectl get ing  ingress  -o jsonpath='{.status.loadBalancer.ingress[*].ip}' --kubeconfig=" + kubeconfig
-	res, err := RunCommand(cmd)
+
+func GenKubeConfigFile(serverName string) (string, error) {
+	cmd := fmt.Sprintf("vagrant ssh %s -c \"cat /etc/rancher/k3s/k3s.yaml\"", serverName)
```

Comment:
```suggestion
	cmd := fmt.Sprintf("vagrant ssh %s -c \"sudo cat /etc/rancher/k3s/k3s.yaml\"", serverName)
```
I think this needs to be sudo because that file is permission restricted for the default vagrant user